src/main/scala/beam/agentsim/agents/ridehail/RideHailManager.scala

```diff
@@ -1699,17 +1620,11 @@ class RideHailManager(
     addVehiclesOnWayToRefuelingDepot(vehiclesHeadedToRefuelingDepot)
     vehiclesHeadedToRefuelingDepot.foreach {
       case (vehicleId, _) =>
-        vehicleManager.putOutOfService(vehicleManager.getRideHailAgentLocation(vehicleId))
+        doNotUseInAllocation.add(vehicleId)
```

Comment:
Ah, good call. Ideally yes, but there was so much code assuming it was an idle vehicle, it seemed easiest to keep it idle and then put out of service once it arrives at parking stall. But I forgot that last part so just put it into a new commit.

Reply:
So the behavior for now is that a vehicle can be interrupted while on the way to a charger... this could still cause problems now I think of it.... hmm

Reply:
ok, think I found the right place to do this... after route created but before dispatching the vehicle. pushing soon